addons/editor-dark-mode/addon.json

```
@@ -1188,6 +1192,10 @@
     "type": "palette",
     "colors": ["primary", "menuBar", "page", "accent", "workspace", "selector"]
   },
+  "latestUpdate": {
+    "version": "1.32.0",
+    "newSettings": ["popup"]
```

```suggestion
    "newSettings": ["popup"],
    "temporaryNotice": "You can now customize the backdrop color of popups, such as the create new variable prompt."
```

My wording is probably bad, but I think it would be worth to explain what the new setting does. In the future this will be explained with setting descriptions.

---

Could we go with this?
```suggestion
    "newSettings": ["popup"],
    "temporaryNotice": "You can now customize the color of popup backdrops (the solid-color background behind prompts for actions like naming a new variable) separately."
```